### PR TITLE
fix EditingRedPacket UI issue in keyboard

### PR DESCRIPTION
**Pull request by jk234ert**
Modification:

1. Adjust inset of `EditingRedPacketViewController's` tableview in keyboard
2. Add separate view for user to choose a wallet in the keyboard, since previous implementation using UIPickerView is invalid in keyboard extension

### CODE REVIEW

**Comment by MainasuK** [comment]
The Red Packet entry may be hide after the user selects a contract. 
(Design detail missing or not applicable)
@neruthes 




- After tapping the "Example" contract cell. The user could tap the "Red Packet" key icon to display the send red packet view panel. But maybe that not expected behavior.
- Previous we only have the encryption option here. And the "Red Packet" icon reuses the "signing" icon which deprecated before. Needs a new icon or refresh entry design. The keyboard should be reflected in Figma.

File: TesserCube/Scene/RedPacket/EditingRedPacket/EditingRedPacketViewController.swift [hunk 1]
```diff
@@ -207,6 +207,11 @@ class EditingRedPacketViewController: UIViewController {
             navigationItem.leftBarButtonItem = UIBarButtonItem(barButtonSystemItem: .stop, target: self, action: #selector(EditingRedPacketViewController.closeBarButtonItemPressed(_:)))
         }
         #endif
+        
+        #if TARGET_IS_KEYBOARD
+        tableView.contentInset = UIEdgeInsets(top: 0, left: 0, bottom: sendRedPacketButton.intrinsicContentSize.height, right: 0)
```
Comment (MainasuK) [inline TesserCube/Scene/RedPacket/EditingRedPacket/EditingRedPacketViewController.swift]: The send button design for in-app using. In the keyboard, we just hide that button could be ok.

File: TesserCube/Scene/RedPacket/EditingRedPacket/RedPacketWalletSelectViewController.swift [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+protocol RedPacketWalletSelectViewControllerDelegate: class {
+    func redPacketWalletSelectViewController(_ viewController: RedPacketWalletSelectViewController, didSelect wallet: WalletModel)
+}
+
+class RedPacketWalletSelectViewController: UIViewController {
+    
+    let tableView: UITableView = {
+        let tableView = UITableView(frame: .zero, style: .grouped)
+        tableView.register(SelectWalletTableViewCell.self, forCellReuseIdentifier: String(describing: SelectWalletTableViewCell.self))
```
Comment (MainasuK) [inline TesserCube/Scene/RedPacket/EditingRedPacket/RedPacketWalletSelectViewController.swift]: Due to reusing system cells with the custom identifier. The custom cell register is not redundant.

**Comment by neruthes** [comment]
Figma link: https://www.figma.com/file/BlCiZUHohnjLqXpdbDmFgi/Keyboard-v2?node-id=0%3A1

👀

**Pull request merged**